soc/bouffalolab/Kconfig.soc

```diff
@@ -0,0 +1,11 @@
+# Copyright (c) 2021-2025 ATL Electronics
+#
+# SPDX-License-Identifier: Apache-2.0
+
+config SOC_FAMILY_BFLB
+	bool
+
+config SOC_FAMILY
+	default "bouffalolab_bflb" if SOC_FAMILY_BFLB
```

Comment:
text does not match, either: `	default "bouffalolab_bflb" if SOC_FAMILY_BOUFFALOLAB_BFLB` or `	default "bflb" if SOC_FAMILY_BFLB` - it must also match the text in the soc.yml file

Reply:
I think I fixed